crates/air-lib/interpreter-data/src/executed_state/impls.rs

```diff
@@ -109,8 +109,12 @@ impl ApResult {
 }
 
 impl CanonResult {
-    pub fn new(cid: Rc<CID<CanonResultCidAggregate>>) -> Self {
-        Self(cid)
+    pub fn executed(cid: Rc<CID<CanonResultCidAggregate>>) -> Self {
+        CanonResult::Executed(cid)
+    }
+
+    pub fn send_request_by(peer_id: Rc<String>) -> Self {
```

Comment:
Mb rename this to `request_sent_by` to by in sync with the `executed` ctor name?